src/session.rs

```diff
@@ -16,6 +16,8 @@ use crate::interpreter_server::{
 pub enum PollNotification {
     Add(VarIdent, Value),
     Remove(VarIdent, Value),
+    FinishedSuccessfully,
+    FinishedWithError(String),
```

Comment:
Add a FIXME here:

`// FIXME: Replace String with InterpretError`

This is easier said then done, because `InterpretError` can possibly contain a dynamic boxed runtime error, which is not currently clone-able, and I don't want to bother you with it.

Reply:
That was my first try, until adding all the Clone traits brought me to the Box and there Rust happened.